packages/website/docs/sources.md

```diff
@@ -288,11 +282,17 @@ See [source](#source) for what to return.
 
 Each source implements the following interface:
 
+### `sourceId`
+
+> `string`
+
+Identifier for the source. It is used as value for the `data-autocomplete-source-id` attribute of the source `section` container.
```

Comment:
Since we're going to enforce uniqueness, we might as well document it now.

```suggestion
Unique identifier for the source. It is used as value for the `data-autocomplete-source-id` attribute of the source `section` container.
```